tools/buildutils/cw/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable AS base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update -y && apt upgrade -y
+RUN apt install -y devscripts config-package-dev debhelper-compat golang protobuf-compiler nasm
```

Comment:
some of these packages aren't needed here since they're specified as build time dependencies of the packages being built and should be installed by the script that builds them.